polyfills.js

```diff
@@ -1,4 +1,8 @@
 // Support for IE
 // https://stackoverflow.com/a/53332776/5594539
+import smoothscroll from 'smoothscroll-polyfill'
```

Comment:
This post saids that you do not need anything because `NodeList.prototype.forEach` is polyfilled by core.js. However, there is nothing about smooth scroll.

Previously we agreed that we do not really care about IE. This polyfill adds behaviour to Edge and Safari but this is not a big deal too.

I would try to avoid CSS polyfills unless it's really necessary but this is up to you. Just keep in mind that adding any extra JS harms user's performance. Not everybody uses machines with high level performance as like as we do.

Reply:
The StackOverflow comment is for code bellow. (I will move it closer to relevant code.) Nonetheless, it seems polyfill `forEach` is not needed. I experienced troubles with it in the first stages of Reader. Apparently it has been fixed in new versions of `core-js` or `pdfjs`.


Smooth scroll polyfill is needed otherwise app fails when `related papers` link is clicked. It could be handled with `try/except block` but I like to have smooth scroll there anyway.

```
The above error occurred in the <PDFRecommender> component:
    in PDFRecommender (created by PDFViewer)
    in div (created by PDFViewer)
    in PDFViewer (created by MainArea)
    in Unknown (created by MainArea)
    in div (created by MainArea)
    in MainArea (created by CoreReader)
    in div (created by Layout)
    in Layout (created by CoreReader)
    in GlobalProvider (created by CoreReader)
    in CoreReader (created by ForwardRef(LoadableComponent))
    in ForwardRef(LoadableComponent) (created by Reader)
    in Reader (created by WithAnalytics)
    in WithAnalytics (created by App)
    in App
    in Container (created by AppContainer)
    in AppContainer

React will try to recreate this component tree from scratch using the error boundary you provided, Reader.
dll_01ec57fc9b90d43b98a8.js (22151,5)

SCRIPT438: SCRIPT438: Object doesn't support property or method 'scroll'
```

Reply:
https://github.com/oacore/reader/pull/89/commits/68ba3bc7fc69facc938047c4453ca405af11d6d3